src/TransactionalMigrationInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Db\Migration;
+
+/**
+ * Allowing to use transaction in migration.
+ */
+interface TransactionalMigrationInterface extends MigrationInterface, RevertibleMigrationInterface
```

Comment:
Suggest use `TransactionalMigrationInterface` without extends, because it's allow create non-revertiable transactioanl migration.

Reply:
In general they could be reverted, they may not be reverted in `MySQL` or `MariaDB`.

Reply:
I means case when we create migration with UP only and want make this transcational:

```php
class MyMigration extends MigrationInterface, Transactionalnterface
```

Reply:
fixed.